static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -90,28 +90,31 @@ export const getEpicTestToRun = memoize(
                     // send ~ one in ten to reduce initial volume
                     if (Math.random() < 0.1) {
                         const page = config.get('page');
-                        const countryCode = geolocationGetSync();
-                        compareVariantDecision({
-                            targeting: {
-                                contentType: page.contentType,
-                                sectionName: page.section,
-                                shouldHideReaderRevenue:
-                                    page.shouldHideReaderRevenue,
-                                isMinuteArticle: config.hasTone('Minute'),
-                                isPaidContent: page.isPaidContent,
-                                tags: buildKeywordTags(page),
-                                countryCode,
-                                showSupportMessaging: !shouldNotBeShownSupportMessaging(),
-                                isRecurringContributor: isRecurringContributor(),
-                                lastOneOffContributionDate: getLastOneOffContributionDate(),
-                                mvtId: getMvtValue(),
-                                epicViewLog: getViewLog(),
-                            },
-                            expectedTest: result ? result.id : '',
-                            expectedVariant: result
-                                ? result.variantToRun.id
-                                : '',
-                        });
+                        // Only compare variants for Epics served in Articles
+                        if (page.contentType === 'Article') {
```

Comment:
Minor, but I'd be tempted to test this first and exit early (just to avoid indenting everything/readability). E.g.

    if (page.contentType !== "Article) {
        return result;
    }

before the if (Math.rand..) check?

Reply:
I thought this would be easier to read with nested ifs, rather than returning early, just for consistency with the other ifs in that block. But it's so so subtle I've updated as suggested.